scss/main.scss

```diff
@@ -14,9 +15,11 @@
 @import '../node_modules/bootstrap/scss/grid.scss';
 @import '../node_modules/react-select/scss/default.scss';
 
-// mixins
+// Foundation Styles
 
-@import 'mixin.scss';
```

Comment:
The /foundation/mixins.scss file doesn't seem to contain everything that this file did. Does everything still work properly/can we get rid of the mixin.scss file?

Reply:
Everything still works properly, this mixin file is the original pulse mixin file. I just moved it up to line 2 ↗️ to separate it from mofo-bootstrap & foundation styles.

Reply:
Wow, I totally missed that.